src/components/DateTimePicker/DateTimePicker.js

```diff
@@ -1,17 +1,22 @@
-import React, { useState, useCallback, useEffect } from 'react';
-import throttle from 'lodash.throttle';
+import React, { useState, useRef } from 'react';
 import { useTranslation } from 'react-i18next';
 import Picker from 'react-datepicker';
 import PropTypes from 'prop-types';
-import 'react-datepicker/dist/react-datepicker.css';
+import './stylesheets/datepicker.scss';
 import en from 'date-fns/locale/en-US';
 import ru from 'date-fns/locale/ru';
 import hy from 'date-fns/locale/hy';
+import checkMobile from 'ismobilejs';
 import CustomInput from './CustomInput';
 
 import { DATE_FORMATS, TIME_FORMATS } from '../../config';
 
-const SMALL_SCREEN_BREAKPOINT = 1280;
+const hiddenInput = {
```

Comment:
Oops? Is there a more elegant way to do this? e.g. setting alpha to 0?

Reply:
i would love to use display:none, but it would make the mobile keyboard disappear as well, making the element useless. other options hide the element, but keep an empty box in its place, which will disrupt the form design. this technique is often used when some element needs to be accessible for screen-readers, but hidden from eye. in our case we dont want screen-readers to see it either, so i added aria-hidden=true. also tab-index=-1, to prevent focusing with keyboard.